rules_gapic/bazel/src/test/java/com/google/api/codegen/bazel/BuildFileGeneratorTest.java

```diff
@@ -46,8 +48,11 @@ public void testRegeneration() throws IOException, InterruptedException {
         .start()
         .waitFor();
 
+    String buildozerPath = getBuildozerPath();
     Path copiedGoogleapis = Paths.get(tempDirPath.toString(), "googleapis");
-    ArgsParser args = new ArgsParser(new String[] {"--src=" + copiedGoogleapis.toString()});
+    ArgsParser args =
+        new ArgsParser(
+            new String[] {"--buildozer=" + buildozerPath, "--src=" + copiedGoogleapis.toString()});
```

Comment:
toString() is called automatically for all objects participating in string cancatenation. Please remove it (here and below)